src/components/RadioGroup/RadioGroup.tsx

```diff
@@ -99,6 +99,7 @@ const RadioGroupItem = ({
         id={id ?? defaultId}
         disabled={disabled}
         required={required}
+        aria-label={`${label}`}
```

Comment:
This could be `aria-label={label}`

Reply:
It won't accept this as aria-label can't handle ReactNode